syncthing/Caddyfile

```diff
@@ -0,0 +1,16 @@
+example.com {
+        gzip
+
+	# redirect all requests (include trailing '/')
+        redir /syncthing /syncthing/ 302
```

Comment:
This redirection should be permanent. The site is broken (missing styling and other assets) without it. Remove the 302 here (implicit 301) or change it to an explicit 301.